src/External/GeneratorPCF.ts

```diff
@@ -70,7 +70,7 @@ export class GeneratorPCF {
         let questionInitSolution: vscode.QuickPickOptions = Placeholders.GetQuickPickOptions(Placeholders.Init_Solution);
         let answerInitSolution: string | undefined = await vscode.window.showQuickPick(["Yes", "No"], questionInitSolution);
 
-        if (!answerInitSolution && answerInitSolution === "Yes") {
```

Comment:
Any specific reason you removed `!answerInitSolution` from the if block? the showQuickPick is returning undefined as well.

Reply:
It was false no matter what you in the prompt. Now when you select YES it will go inside IF and both NO & UNDEFINED will result in skipping the solution part.